### PR TITLE
feat: added version endpoint to check vince version

### PR DESCRIPTION
**Pull request by ankitsridhar16**
Added an endpoint to check installed vince version. 
You can now use `curl  "http://localhost:8080/version"  ` to check the version
the result is
`{"version": "v0.0.67"}`

### CODE REVIEW

File: internal/cluster/http/service.go [hunk 1]
```diff
@@ -985,6 +987,16 @@ func (s *Service) handleReady(w http.ResponseWriter, r *http.Request, params Que
...
 }
 
+func (s *Service) handleVersion(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte(`{"version": "` + version.VERSION + `"}`))
```
Comment (gernest) [inline internal/cluster/http/service.go:997]:
```suggestion
	s.write(w, &v1.Version{
		Version: version.VERSION,
	})
```

**Review by gernest — CHANGES_REQUESTED** [review]
Looks good, since we chose to use json, I left a suggestion to use structured version proto that we already have.

**Comment by ankitsridhar16** [comment]
I have applied your suggestions @gernest

**Review by gernest — APPROVED** [review]
Thanks, LGTM

**Pull request merged**